css/Base/_colors.scss

```diff
@@ -0,0 +1,9 @@
+$color-white: #fff !default;
+$color-black: #000 !default;
```

Comment:
warto dodawac "!default" tylko tym zmiennym ktore potem faktycznie nadpisujesz, daje to troche wieksza kontrole przynajmniej na poczatkowym etapie z themami, aczkolwiek tak jak zrobiles tez jest ok, tylko zeby Ci sie potem nie mylilo ze do kazdej nowo stworzonej zmiennej musisz to dodawac